include/utilities/format/bp1/BP1Writer.h

```diff
@@ -527,4 +282,6 @@ class BP1Writer : public BP1Base
 } // end namespace format
 } // end namespace adios
 
+#include "utilities/format/bp1/BP1Writer.tcc"
```

Comment:
Once included in .cpp....how are the .tcc contents visible when calling the functions in .h?